src/lib/include/libsed.h

```diff
@@ -13,6 +13,8 @@
 
 #define SED_MAX_KEY_LEN (32)
 
+#define SED_OPAL_MAX_LRS 9
```

Comment:
this is confusing to: here SED_OPAL_MAX_LRS define is set to 9 which suggests to me that it is reflecting minimal required total number of ranges (spec: at least 8 locking ranges + 1 global range).
Below in array definition you add 1. Can you explain why?

Reply:
I can't find any reason for this so I removed this one additional LR.